Documentation/developers/debugging.rst

```diff
@@ -193,7 +193,186 @@ If you use CLion, you can debug Polycube with the following steps:
 3. At this point there are two equivalent ways to debug Polycube with CLion:
 
    a. Use the CLion debugger: at the top right select Debug 'polycube';
-   b. (or) Run Polycube in a terminal like that ``sudo polycubed --loglevel = DEBUG`` in order to enable the Polycube Debug logs and then from CLion, go to Run->Attach to Process.. and search for "polycubed"
+   b. (or) Run Polycube in a terminal with enabled debug logs (e.g., ``sudo polycubed --loglevel = DEBUG``); then, from CLion, go to Run->Attach to Process.. and search for "polycubed"
+
+4. At this point, from another terminal, just use ``polycubectl`` to interact with Polycube.
+
+
+
+Debugging network traffic using tcpdump/Wireshark
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Debugging (i.e., sniffing) network traffic frlowing *between* two Polycube services (e.g., on the link that connects a Polycube router to a Polycube bridge) can be achieved with the :ref:`span mode <span-mode>`.
+Please refer to that section for more details and for some working examples.
+
+
+Let's see how to capture packets using the concepts of shadow port and :ref:`span mode <span-mode>` to see what happens. 
```

Comment:
@pinoOgni 
1) The real value of a span port is when you have to intercept the traffic from a Polycube service to another.
I kindly ask you to update the example using such a model, e.g., by connecting two simplebridges and showing how to capture the traffic in between.
2) After looking at the documentation, I feel this text should be moved to the other file where we present the "span" mode. I leave you to make this change.
Thanks!

Reply:
1. Perfect, I will update it in the next few days
2. So I move everything I wrote here in the `debugging` file, in the new section "Traffic debugging with Span Mode" in the `cubes` file, correct?